__tests__/app.test.js

```diff
@@ -147,6 +155,14 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200: Responds with an empty array when no articles match the author filter", () => {
+    return request(app)
+    .get("/api/articles?author=lurker")
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.articles).toEqual([])
+    })
+  })
```

Comment:
This isnt a query that should be added for this specific task. If you wish to add extra behaviours please create a separate branch for them and that is something you wont need to sent through to us, but I would recommend doing this after completing all of the tasks first